preview/app/AppLoader.scala

```diff
@@ -1,26 +1,82 @@
+import commercial.controllers.CommercialControllers
+import common.Logback.LogstashLifecycle
+import common.dfp.FaciaDfpAgentLifecycle
+import conf.{CachedHealthCheckLifeCycle, FootballLifecycle}
+import conf.switches.SwitchboardLifecycle
 import contentapi.{CapiHttpClient, ContentApiClient, HttpClient}
-import controllers.{HealthCheck, ResponsiveViewerController, StandaloneControllerComponents}
-import http.StandaloneFilters
+import controllers._
+import controllers.front.FrontJsonFapiDraft
+import cricket.conf.CricketLifecycle
+import cricket.controllers.CricketControllers
+import dev.DevAssetsController
+import feed.OnwardJourneyLifecycle
+import football.controllers.FootballControllers
+import http.PreviewFilters
 import model.ApplicationIdentity
 import play.api.ApplicationLoader.Context
-import play.api.BuiltInComponentsFromContext
+import play.api.{BuiltInComponents, BuiltInComponentsFromContext}
 import play.api.http.HttpErrorHandler
+import play.api.libs.ws.WSClient
 import play.api.mvc.EssentialFilter
 import play.api.routing.Router
 import router.Routes
-import services.OphanApi
+import rugby.conf.RugbyLifecycle
+import rugby.controllers.RugbyControllers
+import services.{ConfigAgentLifecycle, OphanApi}
 
-class AppLoader extends FrontendApplicationLoader {
-  override def buildComponents(context: Context): FrontendComponents = new BuiltInComponentsFromContext(context) with AppComponents
+trait PreviewLifecycleComponents extends SportServices with CommercialServices with FapiServices with OnwardServices {
+  self: FrontendComponents =>
+
+  //Override conflicting members
+  override lazy val capiHttpClient: HttpClient = wire[CapiHttpClient]
+  override lazy val contentApiClient = wire[ContentApiClient]
+  override lazy val ophanApi = wire[OphanApi]
+
+  def standaloneLifecycleComponents: List[LifecycleComponent] = List(
+    wire[LogstashLifecycle],
+    wire[CommercialLifecycle],
+    wire[OnwardJourneyLifecycle],
+    wire[ConfigAgentLifecycle],
+    wire[FaciaDfpAgentLifecycle],
+    wire[SwitchboardLifecycle],
+    wire[FootballLifecycle],
+    wire[CricketLifecycle],
+    wire[RugbyLifecycle]
+  )
+}
+
+trait PreviewControllerComponents
+  extends ApplicationsControllers
+    with AdminJobsControllers
+    with ArticleControllers
+    with CommercialControllers
+    with FaciaControllers
+    with OnwardControllers
+    with FootballControllers
+    with CricketControllers
+    with FrontendComponents
+    with RugbyControllers {
+  self: BuiltInComponents =>
+
+  def wsClient: WSClient
+  def frontJsonFapiDraft: FrontJsonFapiDraft
+
+  lazy val assets = wire[Assets]
+  lazy val devAssetsController = wire[DevAssetsController]
+  lazy val emailSignupController = wire[EmailSignupController]
+  lazy val faciaDraftController = wire[FaciaDraftController]
+  lazy val faviconController = wire[FaviconController]
+  lazy val itemController = wire[ItemController]
+  lazy val oAuthLoginController = wire[OAuthLoginPreviewController]
 }
```

Comment:
I kept the lifecycle and controllers component traits in the same file, rather than having separate files. This is the only place they are actually used